src/components/MapBox.tsx

```diff
@@ -119,61 +134,46 @@ export const MapBox: React.FC = () => {
 
   // to highlight a given event marker use url params "focusedEvent"
   const [searchParams] = useSearchParams();
-  const focusedEvent = useMemo(() => searchParams.get('focusedEvent'), [searchParams]);
+  const focusedEvent = useMemo(
+    () => searchParams.get('focusedEvent') ?? '',
+    [searchParams]
+  );
 
   // TODO: replace this with activeAccommodations
+  // apply a callback function to transform returned accomodation objects
   const { accommodations, coordinates, isLoading, latestQueryParams, isFetching, error } =
-    useAccommodationsAndOffers();
+    useAccommodationsAndOffers(
+      undefined,
```

Comment:
instead of passing a second prop, it is maybe a bit cleaner to pass it as part of the `SearchProps` object. 

Then you don't have to pass an `undefined` here.

Reply:
Besides breaking the logic for `useQuery`, adding this to `SearchProps` will require making all other fields of `SearchProps` optional for cases where no other fields are available. Made the parameter into an object instead with a separate field for `accommodationtransformFn`